.github/workflows/publish_snapshot.yml

```diff
@@ -66,5 +66,6 @@ jobs:
           echo "</server></servers></settings>" >> $tmp_settings
           
           mvn --settings $tmp_settings clean deploy -Dgpg.skip -Drat.skip -DskipTests -Papache-release,spark3,flink1
+          mvn --settings $tmp_settings clean deploy -Dgpg.skip -Drat.skip -DskipTests -Papache-release,spark3,scala_2.13,flink1
```

Comment:
Do we need to publich 2.13? @YannByron @JingsongLi

Reply:
I think we need to publich it, because if other modules are compiled using scala 2.13 and reference the `paimon-spark` module, and the `paimon-spark` module only supports scala 2.12. other modules will fail to compile.